src/renate/benchmark/models/vision_transformer.py

```diff
@@ -119,6 +144,7 @@ def __init__(self, **kwargs: Any) -> None:
             num_heads=12,
             hidden_dim=768,
             mlp_dim=3072,
+            pretrained_name="google/vit-base-patch16-224",
```

Comment:
why still pass the other arguments? they aren't used, right? or is this a requirement for reloading the model?

Reply:
Point.

Reply:
Ok, I thought the idea was to reload it without `pretrained_name`. Now, it will not be able to reloaded since `pretrained_name` is missing? Can we maybe add a test for this model or do we already have that?

Reply:
`model_fn` loads through `from_state_dict` if `model_state_url` is specified. So reloading (in principle) should be able to restore the weights.

Reply:
What would a test for this do?